<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-5DP82P4" ignored="0" name="Test1" tests="1" failures="1" timestamp="01 Dec 2019 07:22:32 GMT" time="157.866" errors="0">
  <testcase name="logintest" time="57.269" classname="testCases.TC_LoginTest_001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_LoginTest_001.logintest(TC_LoginTest_001.java:39)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
</testsuite> <!-- Test1 -->
